packages/cli/src/templates/app-template.js

```diff
@@ -17,6 +20,76 @@ import './list';
 connectRouter(store);
 
 class AppComponent extends LitElement {
+
+  async connectedCallback() {
+    super.connectedCallback();
+    const route = window.location.pathname;
+    const response = await Promise.all([
+      await client.query({
+        query: ConfigQuery
+      }),
+      await client.query({
+        query: GraphQuery
+      })
+    ]);
+    const { config } = response[0].data;
+    const currentPage = response[1].data.graph.filter((page) => {
+      return route === page.link;
+    })[0];
+    const currentPageTitleSuffix = currentPage.link === '/'
+      ? ''
+      : ` - ${currentPage.title}`;
+    const fullTitle = `${config.title}${currentPageTitleSuffix}`;
+
+    this.setDocoumentTitle(fullTitle);
+    this.setMeta(config.meta, currentPage);
+  }
+
+  setDocoumentTitle(title = '') {
+    const head = document.head;
+    const titleElement = head.getElementsByTagName('title')[0];
+
+    titleElement.innerHTML = title;
+  }
+
+  setMeta(meta = [], currentPage = {}) {
```

Comment:
I really liked this in a separate component as people will want to do custom app-templates, it would be easier to simply import the metaComponent.  Thoughts?